### ISSUE TITLE
I miss ClaimChunk...

### ISSUE THREAD

**Issue by MrmaximoExcssum** [cjburkey01/ClaimChunk#202]
This is a bit of a personal request but would love if you where to do it <3

Could u fix a bug in official "ClaimChunk 0.0.22 release" where ActionBar dont work with paper 1.17.1 and send me the file?

**Why cant i just wait untill 0.0.23 is released?**
The reason is that people have ruined ClaimChunk by suggesting many really dumb ideas that have been added in the last snapshots.

**Comment by Geolykt** [comment]
I do not think that it has to do anything with the influx of new updates but rather with the plugin being written on an (now) unstable codebase

**Comment by cjburkey01** [comment]
> I do not think that it has to do anything with the influx of new updates but rather with the plugin being written on an (now) unstable codebase. Tough I guess it being still in the 0.x.x phase should give you an idea that this is bound to happen

^^^ yes, ClaimChunk wasn't actually ever released as stable, hence the pre-release markings. It's not that I haven't put enough work into it for it to be stable, it's that I've changed my mind too many times along the way and migrating the codebase has made my changes far less than transparent for the 0.0.23 snapshots (which is why I haven't released it yet, I won't until 0.0.22 configs can automatically and reliably convert as well as ensuring some current bugs are fixed).

I understand your sentiment, but Goldmensch, T0biii, Geolykt, and I have worked to bring different updates and replacements to back-ends, they just aren't stable in these snapshots because I've been *really* slow (life has been busy on my end) and haven't gotten it polished up to my own standards. There are massive amounts of virtually untested code running on people's servers because they're willing to test the new snapshots out (shout-out to the 5%-10% of users who are running unstable versions to help me spread out my bug testing reach) in the meantime to make sure what changes have occurred do work.

I will also agree that the world profiles were targeted more towards users who wanted customization and their implementation cut off a large portion of the user-base, being people who wanted to be able to have a simple plugin that just works. Aside from getting the commands rewritten, I haven't worked yet on many things to make it easier to get the plugin setup. Once I have though, I think it'll come together to be a little easier than relying on somewhat ambiguous config options.

Basically, my point is don't compare the current 0.0.23 snapshots to 0.0.22 because they're kind of transitional towards more elegant solutions (which I may delay just a tad to release 0.0.23 as 0.1.0 if I decide to polish ahead). But 0.1.0 would also be relatively incomparable to 0.0.22 because it'd be handled differently, though hopefully not painfully for any kind of users. Ideally, the setup will work out-of-the-box for most users and be configurable in-game via GUI/commands for the rest).
